lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -121,15 +121,17 @@ class JsonpMainTemplatePlugin {
 						: "",
 					"script.charset = 'utf-8';",
 					`script.timeout = ${chunkLoadTimeout / 1000};`,
-					crossOriginLoading
-						? `script.crossOrigin = ${JSON.stringify(crossOriginLoading)};`
-						: "",
 					`if (${mainTemplate.requireFn}.nc) {`,
 					Template.indent(
 						`script.setAttribute("nonce", ${mainTemplate.requireFn}.nc);`
 					),
 					"}",
 					`script.src = ${mainTemplate.requireFn}.p + ${scriptSrcPath};`,
+					crossOriginLoading
```

Comment:
I believe setting `script.crossorigin` after `script.src` is safe because:
1. `script.src` only needs to be set after other attributes because of an issue in IE where script loading will commence immediately after the attribute is set
2. According to this [MDN compat table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#Browser_compatibility) the script `crossorigin` attribute is unsupported in IE, so setting it after `script.src` won't matter.